tests/fsharp/optimize/tests_optimize.fs

```diff
@@ -0,0 +1,276 @@
+module ``FSharp-Tests-Optimize``
+
+open System
+open System.IO
+open NUnit.Framework
+
+open NUnitConf
+open PlatformHelpers
+open FSharpTestSuiteTypes
+
+let testContext = FSharpTestSuite.testContext
+
+module Analyses = 
+
+    let ``fsc >a 2>&1`` cfg dir = 
```

Comment:
i know, but `2>&1` it's the .bat script language, mean redirect standard error (stderr) to standard output (stdout). I'll add a comment, but is there to have 1-1 conversion from .bat script language